_getting-started/index.md

```diff
@@ -7,55 +7,117 @@ includeTOC: true
+The Scala Installer based on coursier will ensure that a JVM and standard Scala tools are installed on your system.  
+
+* Download coursier launcher and execute `setup` command
+
+<div class="main-download">
+    <div id="download-step-one">
+        <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a> then run:</p>
+        <p><code>$ ./cs setup</code></p>
+    </div>
+</div>
+
+
+Along with managing JVMs, it also install useful command-line tools: 
+Ammonite, coursier, scala, scalac, sbt-launcher, scalafmt.
+
+For more information, read [couriser-cli documentation](https://get-coursier.io/docs/cli-overview)
+
+### ...Or manually
+1. if you don't have java 1.8 or 11 installed, download 
+Java from [Oracle Java 8](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html), [
+Oracle Java 11](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html), 
+or [AdoptOpenJDK 8/11](https://adoptopenjdk.net/). Refer JDK Compatibility for Scala/Java compatibility detail.
+1. Install [sbt](https://www.scala-sbt.org/download.html)
+
+
+## Create a Hello-world project with sbt
+Sbt is a build tool for Scala. sbt compiles, runs, 
+and tests your projects among other related tasks.
+
+1. `cd` to an empty folder.
+1. Run the following command `sbt new scala/hello-world.g8`.
+This pulls the 'hello-world' template from GitHub.
+It will also create a `target` folder, which you can ignore.
+1. When prompted, name the application `hello-world`. This will
+create a project called "hello-world".
+1. Let's take a look at what just got generated:
+
+```
+- hello-world
+    - project (sbt uses this to install and manage plugins and dependencies)
+        - build.properties
+    - src
+        - main
+            - scala (All of your scala code goes here)
+                - Main.scala (Entry point of program) <-- this is all we need for now
+    - build.sbt (sbt's build definition file)
+```
+
+More documentation can be found in the [Scala Book](/overviews/scala-book/scala-build-tool-sbt.html) 
+or in the official sbt [documentation](https://www.scala-sbt.org/1.x/docs/index.html)
+
+## Open Hello-world project
+### Using Intellij
+1. Download and install [IntelliJ Community Edition](https://www.jetbrains.com/idea/download/)
+1. Install Scala plugin by following [the instructions on how to install IntelliJ plugins](https://www.jetbrains.com/help/idea/managing-plugins.html)
+1. Open the build.sbt file then choose *Open as a project*
+
+### Using Vscode with metals 
+1. Download [VSCode](https://code.visualstudio.com/Download)
+1. Install the Metals extension from [the Marketplace](https://marketplace.visualstudio.com/items?itemName=scalameta.metals)
+1. Next, open the directory containing a `build.sbt` file. The extension activates when a *.scala or *.sbt file is opened.
+
+## Run Hello World
+Open a terminal
```

Comment:
IntelliJ (and maybe Metals?) lets you run the project by just clicking on a button. In comparison, these instructions are a bit intimidating. Do you think we could include the two approaches?

Reply:
Most people I know prefer to run sbt than using the button "run" in intellij. It doesn't work magically with all projects, sometimes you need to configure correctly the run configuration.
I think it's better to give a solution that always works, and most projects give the "sbt" solution rather than the "run button on intellij". 
I tried on metals too, the run button doesn't work without any configuration.

Reply:
I understand, and I’m one of those who use sbt to run my programs. Not because I prefer doing that, though, but because it doesn’t always work with IntelliJ, as you said.

However, for this getting started example it should work (I didn’t check, though), I think it’s better to show a simpler getting started experience. Then people can learn about sbt when they start working on bigger projects.